elfloader-tool/src/arch-arm/64/crt0.S

```diff
@@ -26,12 +23,13 @@ BEGIN_FUNC(_start)
      * Binary images may not be loaded in the correct location.
      * Try and move ourselves so we're in the right place.
      */
+    mov     x0, #0 /* Arg 0 of fixup_image_base is NULL */
```

Comment:
Isn't there a possibility on ARM also that we get passed a DTB here, and thus we have to check that it's not overwritten?